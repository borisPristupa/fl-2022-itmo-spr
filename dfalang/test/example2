Start  : a
a -> a : a

