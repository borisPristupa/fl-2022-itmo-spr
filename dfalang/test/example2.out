Alphabet: a   
States: a   
Start: a    
Terminals: 
a -> a: a
